_posts/2020-06-03-using-Torch-with-Flux-2.md

```diff
@@ -0,0 +1,116 @@
+# Using Torch kernels inside Flux.jl
+
+With [Flux.jl](https://github.com/Flux/Flux.jl), we have demonstrated how we see a high performance and flexible [differentiable programming](./2019-03-05-dp-vs-rl.md) framework to look like and show its intended use cases to be far reaching and allowing for meaningful speedups. We use it to target highly specialised hardware accelerators like [GPUs](https://fluxml.ai/Flux.jl/stable/gpu/) or [TPUs](https://arxiv.org/pdf/1810.09868.pdf). In our framework, we expect to use as much of the Julia ecosystem, and play nicely with custom types that our users define.
+
+In that spirit, we introduce [Torch.jl](https://github.com/FluxML/Torch.jl), a package that wraps the optimised kernels written for torch and PyTorch, and makes them available to use through Julia. We expect to do this to gain coverage over our existing GPU stack, and bring us plenty of performance improvements on GPUs along the way. As an example we take a popular object detection model - ResNet50 - and compare it with our existing tooling.
+
+![img](https://user-images.githubusercontent.com/20618666/79492160-e60a5600-801f-11ea-9d7f-dd04af243e44.png)
+*Image by github user https://github.com/gartangh*
```

Comment:
I think min/median/etc. is not worthwhile to show. Show the best running times on all, across 3-4 different benchmarks.